companion-app/src/Components/UploadFile.js

```diff
@@ -240,6 +276,38 @@ class UploadFile extends Component {
         this.setState({ echodry: event.target.value });
       }
 
+      applyFlange(event) {
+        this.setState({ flangeapply: event.target.value });
+      }
```

Comment:
This will set the value for apply to either "on" or "off". Check out the applyEq() and applyEcho() functions. You can see if the checkbox is checked or not using ```event.target.checked```. From there you just set the value to true or false.